docs/source/deployment/index.rst

```diff
@@ -1,68 +1,179 @@
 Deployment
 ==========
 
-In the modern age of machine learning, deploying models effectively and consistently plays a pivotal role. The capability to serve predictions 
-at scale, manage dependencies, and ensure reproducibility is paramount for businesses to derive actionable insights from their ML models. 
-Whether it's for real-time predictions, batch processing, or interactive analyses, a robust model serving framework is essential.
+.. important::
 
-MLflow offers a comprehensive suite tailored for seamless model deployment. With its focus on ease of use, consistency, and adaptability, 
-MLflow simplifies the model serving process, ensuring models are not just artifacts but actionable tools for decision-making.
+    This page describes the toolset for deploying your in-house MLflow Model. For information on the **LLM Deployment Server** (formerly known as AI Gateway), please refer to `MLflow Deployment Server <../llms/deployments/index.html>`_.
 
-The Power of MLflow in Model Serving
-------------------------------------
+After training your machine learning model and ensuring its performance, the next step is deploying it to a production environment.
+This process can be complex, but MLflow simplifies it by offering an easy toolset for deploying your ML models to various targets, including local environments, cloud services, and Kubernetes clusters.
 
+.. figure:: ../_static/images/deployment/mlflow-deployment-overview.png
+    :align: center
+    :figwidth: 90%
+
+
+By using MLflow deployment toolset, you can enjoy the following benefits:
+
+- **Effortless Deployment**: MLflow provides a simple interface for deploying models to various targets, eliminating the need to write boilorplate code.
 - **Dependency and Environment Management**: MLflow ensures that the deployment environment mirrors the training environment, capturing all dependencies. This guarantees that models run consistently, regardless of where they're deployed.
-  
 - **Packaging Models and Code**: With MLflow, not just the model, but any supplementary code and configurations are packaged along with the deployment container. This ensures that the model can be executed seamlessly without any missing components.
+- **Avoid Vendor Lock-in**: MLflow provides a standard format for packaging models and unified APIs for deployment. You can easily switch between deployment targets without having to rewrite your code.
```

Comment:
I'm working with a customer migrating from AWS to the Databricks stack. Would it be useful to have a demo for migrating between deployment methods? 
cc @BenWilson2

Reply:
Sounds like a good follow-up! Tho I'm not very sure about the migration process as a whole. Any thoughts @BenWilson2?

Reply:
It's definitely a bit simpler :D I think for Databricks-specific instructions, we're going to have to leave those in the Databricks docs, though. We could provide links :)